solr/solr-ref-guide/src/requesthandlers-and-searchcomponents-in-solrconfig.adoc

```diff
@@ -169,3 +169,15 @@ Many of the other useful components are described in sections of this Guide for
 * `TermVectorComponent`, described in the section <<the-term-vector-component.adoc#the-term-vector-component,The Term Vector Component>>.
 * `QueryElevationComponent`, described in the section <<the-query-elevation-component.adoc#the-query-elevation-component,The Query Elevation Component>>.
 * `TermsComponent`, described in the section <<the-terms-component.adoc#the-terms-component,The Terms Component>>.
+
+Other components that ship with Solr include:
```

Comment:
I'm surprised you put these components in a separate table, rather than appending them to the existing list right above here.  Any particular reason you kept this list separate from the list above?

Reply:
These each have a detailed section in the Ref Guide, and in fact I was thinking of moving the `Get` component, which is covered in the Ref Guide up to here as well.   Having said that, I'm rethinking the way the page lists the components out, and maybe taking a bigger pass through.